packages/create-sitecore-jss/src/templates/nextjs/.env

```diff
@@ -42,6 +42,11 @@ DEFAULT_LANGUAGE=
 # The way in which layout and dictionary data is fetched from Sitecore
 FETCH_WITH=<%- fetchWith %>
 
+# Indicates whether SSG `getStaticPaths` pre-render any pages
+# Set the environment variable DISABLE_SSG_STATIC_PATH_FETCH=true
+# to enable full ISR (Incremental Static Regeneration) flow, pre-fetch static paths 
```

Comment:
Not sure what ", pre-fetch static paths" at the end means. Remove it?
```suggestion
# to enable full ISR (Incremental Static Regeneration) flow
```

Reply:
Wanted to describe a bit which functionality will be disabled, but it's fine to remove it